support/run_template.html

```diff
@@ -14,7 +14,8 @@ <h1>In Browser Testing Page</h1>
 <script type="text/javascript">
 function outf(text) {
    var mypre = document.getElementById(Sk.pre);
-   mypre.innerHTML = mypre.innerHTML + text;
```

Comment:
I'm guessing this HTML-munging was meant to be in a separate PR :)

Reply:
Yeah I guess.